components/shared/logo/logo.tsx

```
@@ -15,6 +15,7 @@ export function Logo({ size = logoSizes.medium, className }: LogoProps) {
         <Link href={routes.home}>
             <Image
                 src={String(logoImage)}
```

Biome (1.9.4)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)